std/fmt/colors.ts

```diff
@@ -156,44 +169,39 @@ export function rgb8(str: string, color: number): string {
 /** Set background color using paletted 8bit colors.
  * https://en.wikipedia.org/wiki/ANSI_escape_code#8-bit */
 export function bgRgb8(str: string, color: number): string {
-  return run(str, code([48, 5, color & 0xff], 49));
-}
-
-function colorToRgbArray(color: Rgb | number): [number, number, number] {
-  if (typeof color === "number") {
-    color = clampAndTruncate(color, 0xffffff);
-    return [
-      (color >> 16) & 0xff, // red
-      (color >> 8) & 0xff, // green
-      color & 0xff, // blue
-    ];
-  } else {
-    return [
-      clampAndTruncate(color.r),
-      clampAndTruncate(color.g),
-      clampAndTruncate(color.b),
-    ];
-  }
-}
-
-/** Set text color using 24bit rgb.
- * You can enter a hexadecimal number like 0xff3280 */
-export function rgb24(str: string, color: Rgb | number): string {
-  return run(str, code([38, 2, ...colorToRgbArray(color)], 39));
-}
-
-/** Set background color using 24bit rgb.
- * You can enter a hexadecimal number like 0xff3280 */
-export function bgRgb24(str: string, color: Rgb | number): string {
-  return run(str, code([48, 2, ...colorToRgbArray(color)], 49));
-}
```

Comment:
Why remove that feature though? With that we can take hexadecimal numbers as input, like `0xff8032`.

Reply:
It should be added in a separate PR

Reply:
Oh I see